book/emails.rst

```diff
@@ -0,0 +1,320 @@
+.. index::
+    :single: E-Mails
+
+Working with E-Mails
+====================
+
+Communicating with customers is an important part of every business. In the Oro Platform, an e-mail
+address is represented by the :class:`Oro\\Bundle\\EmailBundle\\Entity\\Email` activity.
```

Comment:
e-mail address is represented by EmailAddress class, and email instance is represented by Email class